search-app/docs/api.md

```diff
@@ -5627,3 +5630,42 @@ curl -XDELETE \
+
+### <a name="apollo-studio"></a> Navigating Apollo Studio
+Navigating to https://graphql.%CMR-ENV%earthdata.nasa.gov/api on the browser providers an interface through the Apollo Studio services on which the CMR API can be queried. The User interface providers autocomplete, concept schema definitions, and request/response handling using the graphql language.
+
+### <a name="cmr-graphql-programmatic-requests"></a> Programmatic requests to CMR-Graphql
+
+The cmr-graphql API can handle POST requests from external clients (In the graphql query language all requsts must be POST requests). A 'Content-Type' with the value 'application/json' must be specified in the header of the request. The body must be valid JSON and should contain the request information you are looking for. The examples below use the `curl` tool. Integration with cmr-graphql will vary in implementaiton by individual programming languages and frameworks which may implement their own libraries or wrappers for graphql requests. The Apollo Studio providers, under the more info button (elipsis button), a `Copy Operation to cURL` which can be used to copy an equivilant `curl` to the query in the interface.
+
+ __Example of retrieving the count from the first page of collections__
+```
+curl 'https://graphql.%CMR-ENV%earthdata.nasa.gov/api' \ 
+-H 'Content-Type: application/json' \
+-H 'Authorization: Bearer XXXXX' \
+-d '{"query":"{ collections{ count}}"}'```
+
+ __Example of retrieving the concept-ids for services associated to a specified collection__
+ ```
+ curl -XPOST 'https://graphql.%CMR-ENV%earthdata.nasa.gov/api' \
+    -H 'content-type: application/json' \
+    -H 'Authorization: XXXXXX' \
+    -d '{"query":"query ($conceptId: [String]) {\n  collections(conceptId: $conceptId) {\n    items {\n      conceptId\n      services {\n        items {\n          conceptId\n        }\n      }\n    }\n  }\n}","variables":{"conceptId":"C1200000001-PROV1"}}'
+    ```
+
+ __Example of retrieving data and passing graphql variables__
+ ```
+  curl 'https://graphql.%CMR-ENV%earthdata.nasa.gov/api' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer XXXXX' \
+  -d '{"query":"query ($params: CollectionsInput) { collections(params: $params) { items { conceptId } } }","variables":{"params":{"limit":5}}}'
+```
+
+### <a name="cmr-graphql-addtional-information"></a> Additional Information
+The cmr-graphql is maintined seperately from the CMR REST API as such, it is expected that new fields added to CMR concepts will lag behind changes to the CMR REST API. Though these should be periodically updated to match the latest results in the CMR API, requested changes can be made by adding an issue to the cmr-graphql repository on ``` https://github.com/nasa/cmr-graphql/issues```.
```

Comment:
```suggestion
The cmr-graphql is maintained separately from the CMR REST API, it is expected that new fields added to CMR concepts will lag behind changes to the CMR REST API. Though these should be periodically updated to match the latest results in the CMR API, requested changes can be made by adding an issue to the cmr-graphql repository on ``` https://github.com/nasa/cmr-graphql/issues```.
```

It might be useful to convert the graphql title into a link to the repo.